### DISCUSSION TITLE
Configuring CORS for embedding

### DISCUSSION THREAD

**Discussion by gambit071490** [owncast/owncast#2915] [Q&A]
Can I configure Owncast to only allow embeds on certain domains? How?

**Comment by gabek** [comment]
You'd want to do that using your reverse proxy configuration. There you'd have the ability to do any kind of filtering you'd like.

**Reply by gambit071490** [reply]
I mean how exactly to do that in Owncast?

The link below tells how can I embed my livestream into my own site. But how can I prevent others from embeding my livestream into other sites?

https://owncast.online/docs/embed/

**Reply by gabek** [reply]
You wouldn't do that through Owncast. You'd do it through your reverse proxy configuration. I'm not sure what you're using but you'd look up documentation that is something like: https://cgscomputer.com/block-iframes-how-to-stop-your-website-from-being-iframed/
